itenium-socks/src/app/app.component.html

```diff
@@ -1 +1,4 @@
+<div [hidden]="this.env.production === true" id="notProdIndicator">
```

Comment:
fyi

Je kan `this.` in templates weglaten
Ook `=== true` hoeft niet (maar sommigen zijn liever expliciet; ik denk team/project decision)